### PR TITLE
Remove friendships

### PR DESCRIPTION
**Pull request by gingeranyhow**

### CODE REVIEW

**Review by acenine — COMMENTED** [review]
I can send a friend request but I can't accept one, so I haven't been able to test friend deletion.

**Comment by gingeranyhow** [comment]
@acenine - Try again maybe? it's working for me.

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -13,30 +14,54 @@ class Header extends React.Component {
       // username: window.location.pathname.substring(1, window.location.pathname.indexOf('/feed'))
     }
   }
+
+  componentDidMount() {
+    console.log('header remount');
```
Comment (jGlass314) [inline client/src/components/Header.jsx]: Do we want this console.log in here?

File: client/src/components/Header.jsx [hunk 2]
```diff
@@ -13,30 +14,54 @@ class Header extends React.Component {
...
     const profilePath = `/profile/${this.props.name}`;
     const feedPath = '/' + this.props.name + '/feed';
+
+    const hasFriendRequests = false;
+
+    let icon = hasFriendRequests
+      ? <Icon color='yellow' name='users' />
+      : <Icon disabled name='users' />
+
+    let friendRequestList=<span>hi</span>
```
Comment (jGlass314) [inline client/src/components/Header.jsx]: not sure what this is. Are we using it anywhere?

File: database-posgtres/index.js [hunk 3]
```diff
@@ -421,11 +420,58 @@ module.exports = {
       });
   },
 
+  removeFriendship: (userId, friendId) => {
+    console.log('inside remove');
```
Comment (jGlass314) [inline database-posgtres/index.js]: remove console.log

File: database-posgtres/index.js [hunk 4]
```diff
@@ -421,11 +420,58 @@ module.exports = {
...
+
+          return pg('users_friendships')
+            .where('user_id_from', userId)
+            .where('user_id_to', friendId)
+            .where('state', 'request')
+            .limit(1)
+            .del();
+
+        } else if (results === 'friends') {
+          console.log('begin delete', results);
```
Comment (jGlass314) [inline database-posgtres/index.js]: remove console.log

File: database-posgtres/index.js [hunk 5]
```diff
@@ -421,11 +420,58 @@ module.exports = {
...
+          return pg.transaction((trx) => {
+            pg('users_friendships')
+              .where('user_id_from', friendId)
+              .where('user_id_to', userId)
+              .where('state', 'friend')
+              .limit(1)
+              .update({'state': 'request'})
+              .transacting(trx)
+              .then((results) => {
+                console.log('step one done')
```
Comment (jGlass314) [inline database-posgtres/index.js]: remove console.log

**Review by jGlass314 — CHANGES_REQUESTED** [review]
Just a few changes. Console.log statements mostly.

**Review by jGlass314 — APPROVED** [review]

**Pull request merged**